sampleapp/build.gradle

```diff
@@ -0,0 +1,50 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+android {
+    compileSdkVersion rootProject.ext.targetSdkVersion
+
+    defaultConfig {
+        applicationId "com.zendesk.sampleapp"
+        minSdkVersion rootProject.ext.minSdkVersion
+        targetSdkVersion rootProject.ext.targetSdkVersion
+        versionCode 1
+        versionName "1.0"
+
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+
+}
+
+repositories {
+    google()
+    jcenter()
+    mavenCentral()
+    maven {
+        url  "http://dl.bintray.com/outbound/maven"
+    }
+}
+
+dependencies {
+    ext.shouldUseLocalDependency = true
```

Comment:
It would be nice to be consistent with the way we currently do this in Support SDK, but it's something to come back to later. For the moment, this is fine.